doc/rules/operator/new_with_parentheses.rst

```diff
@@ -78,13 +78,28 @@ Rule sets
 
 The rule is part of the following rule sets:
 
-- `@PER <./../../ruleSets/PER.rst>`_
-- `@PER-CS <./../../ruleSets/PER-CS.rst>`_
+- `@PER <./../../ruleSets/PER.rst>`_ with config:
+
+  ``['anonymous_class' => false]``
+
+- `@PER-CS <./../../ruleSets/PER-CS.rst>`_ with config:
+
+  ``['anonymous_class' => false]``
+
 - `@PER-CS1.0 <./../../ruleSets/PER-CS1.0.rst>`_
-- `@PER-CS2.0 <./../../ruleSets/PER-CS2.0.rst>`_
+- `@PER-CS2.0 <./../../ruleSets/PER-CS2.0.rst>`_ with config:
+
+  ``['anonymous_class' => false]``
+
 - `@PSR12 <./../../ruleSets/PSR12.rst>`_
-- `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_
-- `@Symfony <./../../ruleSets/Symfony.rst>`_
+- `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_ with config:
+
+  ``['anonymous_class' => false]``
+
+- `@Symfony <./../../ruleSets/Symfony.rst>`_ with config:
+
+  ``['anonymous_class' => false]``
```

Comment:
have you confirmed with Symfony community that they are OK with this change ?

Reply:
I have not. The documentation was auto updated because Symfony inherits from PER 2.0, so I didn't think much about it. How would you like me to go about getting their confirmation? 

I guess if they're not OK with the change, we'll can add `new_with_parentheses => true` to the Symfony ruleset so there's effectively no change there.

Reply:
ideally check with Symfony folks (eg apply the change and ask for decision as PR to their repo)

less ideally, change the value for symfony ruleset to previous value

Reply:
Symfony has accepted the change in symfony/symfony#57901.

Reply:
fantastic !